Lib/test/test_zipfile.py

```diff
@@ -2392,5 +2394,121 @@ def test_extract_command(self):
                             with open(path, 'rb') as f:
                                 self.assertEqual(f.read(), zf.read(zi))
 
+
+consume = tuple
```

Comment:
Why is this here? Why not just use `tuple` in place?

Reply:
In the external library, this function came from [more_itertools.consume](https://more-itertools.readthedocs.io/en/stable/api.html#more_itertools.consume) (because I can't import it from the stdlib docs). I chose not to inline `tuple` here because I wanted to be explicit about the purpose of this function (consume an iterable), but didn't feel like copy/pasting the recipe. My main motivation for not using it in-place is I don't want someone to wonder "why not `list` or `dict.from_keys()` or `set()` or `all()`?", any of which would work just as well, but none of which are optimal. I'll add a comment.

Reply:
In c8c151113c, I added a comment to that object explaining its purpose. Let me know what you think.

Reply:
Sounds good, and thanks for adding the comment, but maybe it should be a comment and not a docstring? :)